src/Core/Components/RatingDisplay/FluentRatingDisplay.razor

```diff
@@ -0,0 +1,21 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@inherits FluentComponentBase
+
+<fluent-rating-display id="@Id"
+                       class="@ClassValue"
+                       style="@StyleValue"
+                       count="@Count?.ToString("0.00", System.Globalization.CultureInfo.InvariantCulture)"
+                       compact="@Compact"
+                       max="@Max?.ToString("0.00", System.Globalization.CultureInfo.InvariantCulture)"
+                       size="@Size.ToAttributeValue()"
+                       color="@Color.ToAttributeValue()"
+                       value="@Value?.ToString("0.00", System.Globalization.CultureInfo.InvariantCulture)"
```

Comment:
Can the web component handle the rounding? And will it then always be displayed in x.5 even if regional settings would use a ','

Reply:
@vnbaaij It tested on my computer, if you use a comma, the value is not accepted and don't displayed.

```html
<fluent-rating-display value="3.5"></fluent-rating-display>
<br />
<fluent-rating-display value="3,5"></fluent-rating-display>
```
![image](https://github.com/user-attachments/assets/2b5c9a1a-a809-4e45-8a52-bd4b462b5fd0)